### ISSUE TITLE
Autocomplete value can get messy during typing

### ISSUE THREAD

**Issue by charlespockert** [aurelia-ui-toolkits/aurelia-kendoui-plugin#11]
To reproduce try typing 2 characters on the "Basic Usage" test page.

The second character usually appears after the autocomplete has updated the input field.

For example type "ch" and the input will complete with "charlesh"

![image](https://cloud.githubusercontent.com/assets/2189477/11715329/4fffd2a2-9f41-11e5-95dd-a41b8b6df3f9.png)

**Comment by adriatic** [comment]
Confirmed - but luckily this is not Aurelia's problem :-)

You should not echo input characters while there is a match (the exception is of course first character entered because the `input` is blank

**Comment by JeroenVinke** [comment]
So if this is telerik's problem, do we need to keep this issue open / do anything with it?

**Comment by adriatic** [comment]
The problem is ours (Charles' to be precise) as his code should stop echoing code when a match is reached **and** no other candidate for the match exists.

**Comment by charlespockert** [comment]
Not me, I'm not matching anything...

I assume this is something to do with the order of events or the way the input is bound to the Aurelia framework

Will help you guys out soon, sorry just got a lot on at the moment (I believe my company are trying to assassinate me with my workload and its also almost Christmas!)

**Comment by adriatic** [comment]
Thanks for this hint - I made my assessment too quickly, it seems.

**Comment by JeroenVinke** [comment]
So, this has something to do with the `suggest` feature of autocomplete. this is after setting `suggest` to false:

![image](https://cloud.githubusercontent.com/assets/2189477/11717572/b1294170-9f51-11e5-9fc7-0dd03f51d481.png)

**Comment by JeroenVinke** [comment]
this was caused by the default value of the `seperator` property. I don't believe the [docs](http://docs.telerik.com/kendo-ui/api/javascript/ui/autocomplete#configuration-separator), because it says that the default value is empty: ''.

Look at this [dojo](http://dojo.telerik.com/@JeroenVinke/azaYA). If you enable both the `suggest` feature and set the `seperator` property to '' then you get our bug. If you remove the seperator property, or set it to null, then it works fine.

So, I have made our wrapper set the seperator property to be null
